exec/java-exec/src/test/java/org/apache/drill/exec/compile/TestClassTransformation.java

```diff
@@ -68,7 +69,7 @@ public void testJaninoClassCompiler() throws Exception {
   @Test
   public void testJDKClassCompiler() throws Exception {
     logger.debug("Testing JDKClassCompiler");
-    sessionOptions.setOption(OptionValue.createString(OptionType.SESSION, ClassCompilerSelector.JAVA_COMPILER_OPTION, ClassCompilerSelector.CompilerPolicy.JDK.name()));
+    sessionOptions.setOption(OptionValue.createString(OptionType.SESSION, ClassCompilerSelector.JAVA_COMPILER_OPTION, ClassCompilerSelector.CompilerPolicy.JDK.name(), OptionScope.SESSION));
```

Comment:
Hmmm...

We should not need every caller that sets a session option to say that it is setting a session option. Instead, the implementation of `setOption` for the session option manager class should fill in the `OptionScope.SESSION` value.